core/admin/adm.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib import admin
+from core.admin.utils import get_final_display_list
+from core.models.adm import AdmColumn, AdmRecord, AdmRow, AdmChoice
+
+
+@admin.register(AdmColumn)
+class AdmColumnAdmin(admin.ModelAdmin):
+    search_fields = [
+        "name",
+    ]
+
+    def get_list_display(self, request):
+        exclude = ["admrecord"]
+        return get_final_display_list(AdmColumn, exclude)
+
+
+@admin.register(AdmRecord)
+class AdmRecordAdmin(admin.ModelAdmin):
+    search_fields = [
+        "country_programme_report__name",
```

Comment:
Avand in  vedere ca am adaugat ieri `autocomplete admin filters`, nu ar strica sa il pune si aici.